src/HW_2_OOP/HufflepuffStudents.java

```diff
@@ -0,0 +1,31 @@
+package HW_2_OOP;
+
+public class HufflepuffStudents extends HogwartsStudentGeneral{
```

Comment:
```suggestion
public class HufflepuffStudent extends HogwartsStudentGeneral{
```
классы нужно называть в единственном числе